core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -82,6 +82,9 @@ class LogManager(logDirs: Seq[File],
   @volatile private var _currentDefaultConfig = initialDefaultConfig
   @volatile private var numRecoveryThreadsPerDataDir = recoveryThreadsPerDataDir
 
+  // Visible for testing
+  private[log] val partitionsInitializing = new ConcurrentHashMap[TopicPartition, Boolean]().asScala
```

Comment:
Can you add a comment indicating that the value indicates that config was updated while the partition was being initialized?

Reply:
Done.